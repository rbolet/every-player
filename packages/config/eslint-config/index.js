module.exports = {
  extends: ['eslint:recommended', 'plugin:@typescript-eslint/recommended'],
  parser: '@typescript-eslint/parser',
  plugins: ['@typescript-eslint'],
  env: {
    node: true,
    es2021: true,
  },
  parserOptions: {
    ecmaVersion: 'latest',
    sourceType: 'module',
  },
  rules: {
    '@typescript-eslint/no-unused-vars': ['error', { argsIgnorePattern: '^_' }],
    '@typescript-eslint/no-explicit-any': 'warn',
  },
};
