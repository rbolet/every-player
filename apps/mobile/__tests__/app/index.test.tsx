import React from "react";
import renderer from "react-test-renderer";
import { TamaguiProvider } from "tamagui";
import Index from "@/app/index";
import { tamaguiConfig } from "@/tamagui.config";

describe("Index Screen", () => {
  it("renders without crashing", () => {
    const tree = renderer.create(
      <TamaguiProvider config={tamaguiConfig}>
        <Index />
      </TamaguiProvider>
    );
    expect(tree).toBeTruthy();
  });

  it("matches snapshot", () => {
    const tree = renderer.create(
      <TamaguiProvider config={tamaguiConfig}>
        <Index />
      </TamaguiProvider>
    );
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
