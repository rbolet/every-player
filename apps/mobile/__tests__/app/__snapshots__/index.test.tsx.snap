// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index Screen matches snapshot 1`] = `null`;
